### ISSUE TITLE
iothub-explorer: Windows-style line ending causing error on MacOS

### ISSUE THREAD

**Issue by dougseven** [Azure/azure-iot-sdks#149]
I was running into an issue using iothub-explorer on my MacBook. It appears to be an issue with Windows-style line endings (CRLF) instead of Unix style (LS).

On MacOS in Terminal, run:
```
npm install -g iothub-explorer
npm update iothub-explorer

iothub-explorer "HostName=<IOT_HUB_NAME>.azure-devices.net;SharedAccessKeyName=iothubowner;SharedAccessKey=<IOT_HUB_KEY>“ list
```

This results in an error

```
env: node\r: No such file or directory
-bash: list: command not found
```

To solve this on my local machine I converted the line endings using dos2unix

```
brew install dos2unix
find /usr/local/lib/node_modules/iothub-explorer -name "*.js" | xargs sudo dos2unix
```

After that I was able to run the same command above successfully.

This needs to be corrected in source.

**Comment by damonbarry** [comment]
The source code in the repo is fine; it's the publishing process that causes this.  I'll update the npm package as soon as I get the chance, and in the future we'll make updates to our publishing process that prevent this problem from happening.

**Comment by damonbarry** [comment]
I've published "1.0.0-preview.6.a" to npmjs.org.  I tested on Windows, but also opened iothub-explorer.js in a text editor and confirmed it has unix line endings rather than windows.
